javascript/javascript3/week3/app.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="app.css">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="listDivBtn">
+            <button id="listBtn" class="listBtn">A list of your screenshots</button>
```

Comment:
You should use `-` for a phrase/compound word. Also, names should be meaningful.
```suggestion
            <button id="list-btn" class="list-btn">A list of your screenshots</button>
```